lms/djangoapps/learner_dashboard/views.py

```diff
@@ -0,0 +1,52 @@
+"""
+Handles requests for views, returning page frame
+"""
+
+import logging
+from urlparse import urljoin
+
+from django.conf import settings
+from django.contrib.auth.decorators import login_required
+from django.views.decorators.csrf import ensure_csrf_cookie
+from django.http import Http404
+
+from edxmako.shortcuts import render_to_response
+from openedx.core.djangoapps.programs.utils import get_programs_for_dashboard
+from learner_dashboard.models import LearnerDashboardConfiguration
+from student.views import get_course_enrollments
+
+
+log = logging.getLogger(__name__)
+
+
+@login_required
+@ensure_csrf_cookie
+def view_programs(requests):
+    if not LearnerDashboardConfiguration.current().enable_learner_dashboard:
```

Comment:
Sorry to be unclear.  What i meant was, since the `is_learner_dashboard_enabled` property accessor was implemented on the config model, my assumption would have been that we'd use that here.  This gives the benefit of being able to concisely check both "enabled" flags on the config model at once, and is consistent with other similar patterns we use with config models (e.g. CredentialsApiConfig, ProgramsApiConfig).

Conversely if it was intended to only check that single attribute here, I wondered where the property accessor was meant to be used instead.